src/testcafe-runner.js

```diff
@@ -111,13 +111,21 @@ async function runTestCafe ({ projectPath, assetsPath, suite, metrics }) {
       debugOnFail: false,
     });
 
+    const timeoutPromise = new Promise((resolve) => {
+      setTimeout(() => {
+        console.error(`Maximum timeout ${timeoutSec} seconds was exceeded`);
```

Comment:
Nitpick: I would say `Test timed out after ${timeoutSec} seconds`

Reply:
Thank! I fixed that.